src/Esprima/JavascriptParser.cs

```diff
@@ -41,7 +41,7 @@ public Context()
             public bool AllowIn;
             public bool AllowStrictDirective;
             public bool AllowYield;
-            public bool Await;
```

Comment:
even thought might not seem a good name, I think we should keep the naming that JS project has to make diffing and syncing easier

Reply:
No problem, I'll revert it.